solutions/landing-zone-v2/namespaces/config-control.yaml

```diff
@@ -0,0 +1,16 @@
+# Grant GCP role Organization Role Admin to GCP config-control-sa a.k.a yakima
```

Comment:
very valid question. all existing permissions for this SA are defined within the bootstrap. I'm opened to both approach

Reply:
I don't think the bootstrap process can have less permissions than these. We can decide that the additionnal permissions will be managed as config connector yaml. Let me know what you think
```bash
gcloud organizations add-iam-policy-binding "${ORG_ID}" \
  --member="serviceAccount:${SA_EMAIL}" \
  --role=roles/resourcemanager.organizationAdmin \
  --condition=None
  
gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
  --member "serviceAccount:${SA_EMAIL}" \
  --role "roles/editor" \
  --project "${PROJECT_ID}"

gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
  --member "serviceAccount:${SA_EMAIL}" \
  --role "roles/iam.serviceAccountAdmin" \
  --project "${PROJECT_ID}"
```

Reply:
I agree we should manage in config connector yaml as much as possible, we'll have to test and review, it can wait for a 
 future issue if it makes sense.

Reply:
this branch holds the new solution https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/gh286-feat-remove-yakima-permissions-bootstrap/solutions/landing-zone-v2/namespaces/config-control.yaml

Reply:
ok, since we have a `management-namespace` setter to possibly replace config-control in the future, maybe we should rename the `config-control.yaml` to something more generic like `management-namespace.yaml`?